csharp/Directory.Build.props

```diff
@@ -10,8 +10,8 @@
   <!-- AssemblyInfo properties -->
   <PropertyGroup>
     <Product>Apache Arrow library</Product>
-    <Copyright>2018 Apache Software Foundation</Copyright>
-    <Company>Apache</Company>
+    <Copyright>Copyright 2016 The Apache Software Foundation</Copyright>
```

Comment:
The "2016" looks weird, but I'm not a lawyer. Either "2019" or "2016-2019" would look more expected. @wesm Any opinion?

Reply:
I took this from https://github.com/apache/arrow/blob/master/NOTICE.txt#L2. I can change it to whatever folks want.

Reply:
If we have to have a year then 2016-2019 would be it

Reply:
I can drop the year too, if you want...

Reply:
Better fix it to "2016-2019", IMHO. This is what https://www.apache.org/legal/src-headers.html#notice seems to recommend.

Reply:
OK, fixed to 

`<Copyright>Copyright 2016-2019 The Apache Software Foundation</Copyright>`